### ISSUE TITLE
Copy/Paste issues

### ISSUE THREAD

**Issue by Taddo** [giuspen/cherrytree#229]
I found problems with copy/past using shortcut and contextual menu, especially pasting text from cherrytree to othe places; i also found that paste text (to Cherry Tree) copied from a <code></code> tag doesn't work.

**Comment by giuspen** [comment]
I hope to get rid of the copy/paste problems in windows after switching to GTK3 from current obsolete GTK2. About the paste from a code tag problem, please provide a real example.

**Comment by evolighting** [comment]
@giuspen Hi , I meet the problems too , Not only in  windows(10 ,auto updated) but also in feodra(25, gnome-shell , xorg)
It seens that copy and paste inside software is OK, but cross software copy/paste seem to need "wait" for a while,  and only then it would work. 
But most of time it would just work without problem. And this issue is appearing more often after update to version 0.38.

**Comment by Taddo** [comment]
I found the code tag problem on this page as well https://www.digitalocean.com/community/tutorials/how-to-install-the-munin-monitoring-tool-on-debian-8

**Comment by HiImLogan** [comment]
Same problem in 0.38.1 using Windows 10 and Windows 7.

**Comment by giuspen** [comment]
For the problem of copying from cherrytree to other applications, if you find problems please use "copy as plain text" if that is acceptable in your workflow, since that should always work.
About the problem of pasting text between code tag, do you mean that it is not pasted formatted properly or that it is not pasted at all. In my test it is pasted with wrong formatting (I'll try and fix that)

**Comment by thistleknot** [comment]
I'm having copy pasta issues as well.

Either Ctrl-C, or context copy via kb shortcut doesn't work, but for some reason, right clicking and selecting copy works...  so annoying, can't use kb shortcuts which is essential with a PIM

**Comment by giuspen** [comment]
this problem will hopefully go when the porting from pygtk2 to gtkmm3 will be complete

**Comment by BurtGummer** [comment]
same Problem with 0.38.6 on Win 10.

**Comment by paricapala** [comment]
Yeah, same problem.  I cannot get Cherry Tree to Copy/Paste into the terminal.  Let me be plain, this problem destroys what otherwise seems like a great program.  Is there a fix?

**Comment by klhutchins1** [comment]
I just ran into this issue as well on 0.38.8 on Windows 10, text was from powershell prompt. It was working smoothly before.
Tried opening cheery tree in Administrator mode and regular, still couldn't copy some text out
Can Copy into Cherry Tree but not out.
I can get around it by cutting and pasting, and pasting back in sometimes, but somewhat annoying. I need to ensure I copy the next line that I want, and not just the one line or partial line. 

I can open some nodes and some things I can copy out some things I can't.

**Comment by giuspen** [comment]
as temporary workaround when you have the issue try using "copy as plain text" Ctrl+Shift+C

**Comment by abclution** [comment]
> as temporary workaround when you have the issue try using "copy as plain text" Ctrl+Shift+C

version 0.38.11 
Windows 10


Using the keyboard shortcut for copying as plain text just doesn't work for me. In order to copy as plain text I MUST right click and use the menu there.

**Comment by MidnightSeer** [comment]
I have this issue still in cherrytree .38.8
deb http://http.kali.org/kali kali-rolling main non-free contrib

no update available in apt update and apt list --upgradable 

Middle click works fine....only a problem when i c/p form the web browser (mozilla).  The keyboard shortcut only brings up the developer options.

**Comment by oscarandreu** [comment]
Same issue, version 0.99.19 on windows

**Comment by UnluckyLuke** [comment]
Starting CherryTree as admin solved the problem for me.

**Comment by wendychen** [comment]
Right-click on the node > Toggle Read-Only
This helps me.
I accidentally toggled a certain node to read-only mode. As a result, I can't paste content into it, Paste as Plain Text, or edit the content.
